src/data/loader.py

```diff
@@ -63,6 +65,16 @@ def load_event_ids(cls, uniq_id: str | None = None):
                 logger.error(e)
                 cls._invalid_uniq_ids.append(event_id)
 
+    @classmethod
+    def _clean_not_existing_event_database_files(cls, event_uniq_ids: list[str]):
+        to_remove = [
```

Comment:
If you're iterating once, use a generator expression rather than a list comprehension